website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -253,6 +253,15 @@ defaults that are inherited by all services.
   specified, inherits the Envoy default for route timeouts (15s). A value of 0 will
   disable request timeouts.
 
+- `balance_inbound_connections` - The strategy used for balancing inbound connections
+  across Envoy worker threads. Connect's Envoy integration currently supports the
```

Comment:
```suggestion
  across Envoy worker threads. Consul service mesh Envoy integration supports the
```
I think we're trying to get away from referring to "Connect" as an entity unless we're talking about the configuration `connect`.